src/utils/datetime_test.cljs

```diff
@@ -190,76 +193,75 @@
-                                          (get datetime/int->weekday))
-          mock-today                (t.format/unparse (t.format/formatters :date-time-no-ms) curr-time)
-          in-n-days                 #(-> (time-str-to-obj mock-today)
-                                         (t/plus (t/days %)))
-          in-n-minutes              #(-> (time-str-to-obj mock-today)
-                                         (t/plus (t/minutes %)))
-          in-n-hours                #(-> (time-str-to-obj mock-today)
-                                         (t/plus (t/hours %)))
-          mock-tomorrow             (in-n-days 1)
-          mock-in-two-days          (in-n-days 2)
-          mock-in-three-days        (in-n-days 3)
-          mock-in-four-days         (in-n-days 4)
-          mock-in-five-days         (in-n-days 5)
-          mock-in-six-days          (in-n-days 6)
-          mock-in-15-minutes        (in-n-minutes 15)
-          mock-in-1-hour            (in-n-hours 1)
-          mock-in-8-hour            (in-n-hours 8)
-          get-month-day-int         #(int (get-day %))
-          today?                    (fn [mocked curr-time]
-                                      (=
-                                       (t.format/unparse (t.format/formatter "MM:DD") mocked)
-                                       (t.format/unparse (t.format/formatter "MM:DD") curr-time)))
-          tomorrow?                 (fn [mocked curr-time]
-                                      (some #(= %
-                                                (-
-                                                 (int (get-month-day-int mocked))
-                                                 (int (get-month-day-int curr-time))))
-                                            [1 30 29 27]))
-          form-full-date            #(str (get-hh-mm %)
-                                          " " (string/capitalize (get-week-day %))
-                                          " " (get-month-day-int %)
-                                          " " (string/capitalize (get datetime/months (t/month %))))
-          today-date                #(str (get-hh-mm %) " today")
-          tomorrow-date             #(str (get-hh-mm %) " tomorrow")
-          write-date                #(cond (today? % curr-time)    (today-date %)
-                                           (tomorrow? % curr-time) (tomorrow-date %)
-                                           :else                   (form-full-date %))
-          will-unmute-in-1-hour     (remove-msecs (time-coerce/to-string mock-in-1-hour))
-          will-unmute-in-8-hours    (remove-msecs (time-coerce/to-string mock-in-8-hour))
-          will-unmute-in-15-mins    (remove-msecs (time-coerce/to-string mock-in-15-minutes))
-          will-unmute-in-two-days   (remove-msecs (time-coerce/to-string mock-in-two-days))
-          will-unmute-tomorrow      (remove-msecs (time-coerce/to-string mock-tomorrow))
```

Comment:
@ibrkhalil, my suggestion here is not to be necessarily applied in this PR. Unit tests should be easy to read and the expected values should be often hardcoded.

If we set the current cljs-time to a fixed timestamp, then I believe all expected values can be transformed to simple strings and devs will instantly know what this function does and what it produces as a result of any input -> output. I can tell you, I had to start the REPL to know what `format-mute-till` produces.

Reply:
Understood, I don't think it's worth it to compute the expected values if we already know them, And like you just said it adds extra complexity in understanding what the test does.
I am back Monday, Will create a follow up for the refactor.

Reply:
https://github.com/status-im/status-mobile/issues/16458